src/Agent/NewRelic/Agent/Core/Utilization/VendorInfo.cs

```diff
@@ -97,13 +100,10 @@ public VendorInfo(IConfiguration configuration, IAgentHealthReporter agentHealth
             if (_configuration.UtilizationDetectDocker)
             {
 #if NETSTANDARD2_0
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                var dockerVendorInfo = GetDockerVendorInfo(new FileReaderWrapper(), RuntimeInformation.IsOSPlatform(OSPlatform.Linux));
```

Comment:
We might also need to make this call available to .net framework applications so that the ECS metadata checks can be made from .net framework windows containers.

Reply:
I was thinking about that and while it would be possible to run a .NET Framework app, I am not sure how many folks would actually do that.

I will take another pass at the code.

Reply:
Latest commit has the changes to allow ECS checks on Framework.